samples/nrf9160/aws_fota/README.rst

```diff
@@ -40,6 +40,11 @@ The `AWS IoT Developer Guide`_ contains all required information about the Amazo
    A file server instance that hosts the new firmware image is required for this sample.
    You can set up and use your own AWS S3 server bucket to host the firmware described in the `Setting up an AWS S3 bucket`_ section.
 
+Configuration
+*************
+
+|config|
+
```

Comment:
If we are taking this part out of overview, then may be we should fit it in 'setup' section?

Reply:
Yeah, this can work.